packages/crawler/tests/e2e/specs/index.e2e.ts

```diff
@@ -0,0 +1,85 @@
+import { crawlPage } from '../../../src/crawl-websites';
+import * as puppeteer from 'puppeteer';
+import { puppeteerConfig } from '../../../constants';
+import { PdsTestingContext, setContentWithDesignSystem } from '../helpers';
+
+export const testCrawlerWithHtmlAndPrefixes = async (pdsTestingContext: PdsTestingContext): Promise<void> => {
+  const browser = await puppeteer.launch(puppeteerConfig);
```

Comment:
Can't we use `jest-puppeteer` with the setup of our shared package here?

Reply:
Fixed it